### PR TITLE
Improves the module examples/complete

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: examples/complete/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Complete
+
+Configuration in this directory creates RAM&FC on Alibaba Cloud.
```
Comment (Wanghx0991) [inline examples/complete/README.md]: RAM&FC -> `RAM` and `FC`

File: examples/complete/tfvars/01-update.tfvars [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+##############################################################
+shard_count           = 3
+auto_split            = true
+max_split_shard_count = 60
+append_meta           = false
+
+##############################################################
+#variables for alicloud_ram_role
+##############################################################
+ram_role_description = "this is my ram role"
```
Comment (Wanghx0991) [inline examples/complete/tfvars/01-update.tfvars]: ram_role_description = "ram_role_description"
这种命名整体修改下,规范起来

File: examples/complete/tfvars/01-update.tfvars [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+force                = true
+
+##############################################################
+#variables for alicloud_ram_role_policy_attachment
+##############################################################
+fc_service_description = "my tf unit test"
```
Comment (Wanghx0991) [inline examples/complete/tfvars/01-update.tfvars]: fc_service_description = "fc_service_description"

File: terratest/terraform_test.go [hunk 4]
```diff
@@ -20,7 +20,7 @@ func TestTerraformBasicExampleNew(t *testing.T) {
 
 	terraformOptions := &terraform.Options{
 		// The path to where our Terraform code is located
-		TerraformDir: "../example/",
+		TerraformDir: "../examples/",
```
Comment (Wanghx0991) [inline terratest/terraform_test.go]: ../examples/complete

File: terratest/terraform_test.go [hunk 5]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/gruntwork-io/terratest/modules/terraform"
 )
 
-// An example of how to test the simple Terraform module in examples/terraform-basic-example using Terratest.
+// An examples of how to test the simple Terraform module in examples/terraform-basic-examples using Terratest.
```
Comment (Wanghx0991) [inline terratest/terraform_test.go]: An example of how to test the simple Terraform module in examples/complete using Terratest

File: examples/complete/variables.tf [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  default     = "System"
+}
+
+##############################################################
+#variables for alicloud_fc_service
+##############################################################
+variable "fc_service_description" {
+  description = "The specification of module fc service description."
+  type        = string
+  default     = "tf unit test"
```
Comment (Wanghx0991) [inline examples/complete/variables.tf]: fc_service_description

File: terratest/terraform_test.go [hunk 7]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/gruntwork-io/terratest/modules/terraform"
 )
 
-// An example of how to test the simple Terraform module in examples/terraform-basic-example using Terratest.
+// An example of how to test the simple Terraform module in examples/terraform-basic-examples using Terratest.
```
Comment (Wanghx0991) [inline terratest/terraform_test.go]: // An example of how to test the simple Terraform module in examples/complete using Terratest.

File: examples/complete/main.tf [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  }
+}
+
+resource "alicloud_log_store" "default" {
+  project               = alicloud_log_project.default.name
+  name                  = var.log_store_name
+  shard_count           = var.shard_count
+  auto_split            = var.auto_split
+  max_split_shard_count = var.max_split_shard_count
+  append_meta           = var.append_meta
```
Comment (xiaozhu36) [inline examples/complete/main.tf]: 上面这两个资源，可以改用 sls 的module： https://github.com/terraform-alicloud-modules/terraform-alicloud-sls/blob/master/main.tf

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: examples/complete/tfvars/01-update.tfvars [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+##############################################################
+#variables for alicloud_ram_role
+##############################################################
+ram_role_description = "ram_role_description"
+force                = true
+
+##############################################################
+#variables for alicloud_ram_role_policy_attachment
+##############################################################
+fc_service_description = "fc_service_description"
```
Comment (xiaozhu36) [inline examples/complete/tfvars/01-update.tfvars]: 这里面可以改的参数跟 variable中的不一致啊

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: examples/complete/tfvars/01-update.tfvars [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+store_max_split_shard_count = 60
+store_append_meta           = false
+
+##############################################################
+#variables for alicloud_ram_role
+##############################################################
+ram_role_description = "update_ram_role_description"
+force                = true
+
+##############################################################
```
Comment (xiaozhu36) [inline examples/complete/tfvars/01-update.tfvars:32]: document 也是可以修改的吧

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**